services/grafana-loki/0.48.7/grafana-loki-pre-install-jobs/pre-install.yaml

```diff
@@ -0,0 +1,99 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: grafana-loki-pre-install
+  namespace: ${releaseNamespace}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: grafana-loki-pre-install
+  namespace: ${releaseNamespace}
+rules:
+  - apiGroups: ["kustomize.toolkit.fluxcd.io"]
+    resources: ["kustomizations"]
+    verbs: ["get", "patch"]
+  - apiGroups: ["ceph.rook.io"]
```

Comment:
(just making sure) i am assuming this will not fail when ceph crds are not installed on the cluster. correct me if that assumption is wrong.

Reply:
yep these apply cleanly without the Ceph CRDs installed